src/main/java/org/junitpioneer/jupiter/DefaultLocale.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * {@code @DefaultLocale} is a JUnit Jupiter extension to change the value
+ * returned by {@link java.util.Locale#getDefault()} for a test execution.
+ *
+ * <p>The {@link java.util.Locale} to set as the default locale can be
+ * configured using only a language, a language and a country or a language, a
+ * country and a variant. If {@link #variant()} is set, but {@link #country()}
+ * is not, a
+ * {@link org.junit.jupiter.api.extension.ExtensionConfigurationException} will
+ * be thrown. After the annotated element has been executed, the default Locale
+ * will be restored to its original value.</p>
+ *
+ * <p>{@code @DefaultLocale} can be used on the method and on the class level.
+ * If a class is annotated, the configured Locale will be the default Locale
+ * for all tests inside that class. Any method level configurations will
+ * override the class level default Locale.</p>
+ *
+ * @since 0.2
```

Comment:
This will still go into `0.1.x` (right @sormuras?), so `@since 0.1` should be fine.

Reply:
ShipKit only increments `BUILD` in `MAJOR.MINOR.BUILD`. So, yes, it should be `0.1.x` and agree to mention it as `@since 0.1` here.

Reply:
Okay, so no SemVer? This is a new feature, so I expected a minor version bump.

Reply:
You're right. I'm inclined to make the next one `0.2` - what do you think @sormuras? And what file does that go into? (Can't check right now.)

@britter We're doing this "every commit to master triggers a release"-thing (see #87), so we have to make the version bump part of this commit.

Reply:
Edit the `version` in `version.properties` to the next version you want.

https://github.com/junit-pioneer/junit-pioneer/blob/961177d5c4cd99f220870c96658f17a769137807/version.properties#L3

Never(!) set it to a previous version, though.

Reply:
Having said that, increasing the minor version on each commit to `master` should be documented (and enforced?) somewhere.

Reply:
@sormuras Not every commit, though, right? See #113.